src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ProjectSecurityAwarenessScore.java

```diff
@@ -27,25 +28,31 @@
   /**
    * A number of points which are added to a score value if a project has a security policy.
    */
-  private static final double SECURITY_POLICY_POINTS = 5.0;
+  private static final double SECURITY_POLICY_POINTS = 3.0;
 
   /**
    * A number of points which are added to a score value if a project has a security team.
    */
-  private static final double SECURITY_TEAM_POINTS = 8.0;
+  private static final double SECURITY_TEAM_POINTS = 5.0;
+
+  /**
+   * A number of points which are added to a score value if a project uses verified signed commits.
+   */
+  private static final double SIGNED_COMMITS_POINTS = 2.0;
 
   private static final String DESCRIPTION = String.format(
       "The score checks if a project has a security policy and a security team.\n"
           + "If the project has a security policy, then the score adds %2.2f.\n"
-          + "If the project has a security team, then the score adds %2.2f.",
-      SECURITY_POLICY_POINTS, SECURITY_TEAM_POINTS);
+          + "If the project has a security team, then the score adds %2.2f.\n"
+          + "If the project uses verified signed commits, then the score adds %2.2f.",
+      SECURITY_POLICY_POINTS, SECURITY_TEAM_POINTS, SIGNED_COMMITS_POINTS);
 
   /**
    * Initializes a new {@link ProjectSecurityAwarenessScore}.
    */
   ProjectSecurityAwarenessScore() {
     super("How well open-source community is aware about security", DESCRIPTION,
-        HAS_SECURITY_POLICY, HAS_SECURITY_TEAM);
+        HAS_SECURITY_POLICY, HAS_SECURITY_TEAM, USES_VERIFIED_SIGNED_COMMITS);
```

Comment:
Could you please mention in the javadoc of this score that it uses the new feature? Like this:

```
...
<li>if a project uses verified signed commits</li>
...
```